src/components/ChatEntry.js

```diff
@@ -1,22 +1,47 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
-const ChatEntry = (props) => {
+
+const ChatEntry = ({sender, body, timeStamp, liked, id, updateChatData}) => {
+
+  const location = sender === 'Vladimir' ? 'local' : 'remote';
+  const heartColor = liked ? '❤️' : '🤍';
+
+  const onHeartClick = () => {
+    const updatedEntry = {
+      id: id,
+      sender: sender,
+      body: body,
+      timeStamp: timeStamp,
+      liked: !liked,
+    };
+    updateChatData(updatedEntry);
+  };
```

Comment:
See my comment in app.js about only sending `updateChatData` an `id` as a parameter (instead of a constructed updatedEntry object. That would make onHeartClick() look like:
```javascript
const onHeartClick = () => {
   updateChatData(id);
};